Makefile

```diff
@@ -0,0 +1,11 @@
+.DEFAULT_GOAL := build
+.PHONY: clean build
+
+dfareporting_utils.py:
+	curl https://raw.githubusercontent.com/googleads/googleads-dfa-reporting-samples/master/python/v2_7/dfareporting_utils.py > dfareporting_utils.py
```

Comment:
I am just copying this file. I have seen that it equals v2_8 one.
Having a look to the  discovery call (https://github.com/google/google-api-python-client/blob/master/googleapiclient/discovery.py#L192) I see that the argument is the verion of the service. I am a bit confused about this as initially I was thinking it was related to the python version....
What is the proper thought?
The thing is that the googleads-dfa-reporting-samples structure is a bit wierd...